### ISSUE TITLE
Without accent on the linux integrated terminal

### ISSUE THREAD

**Issue by Felipebros** [microsoft/vscode#103168]
- VSCode Version: 1.47.2
- OS Version: Ubuntu 18.04.4 LTS x86_64

Steps to Reproduce:

1. Terminal
2. New terminal
3. type "é um teste de solução" incorrect result "e um teste de soluçao"

Does this issue occur when all extensions are disabled?: Yes

**Comment by Tyriar** [comment]
I can't reproduce, is your `$LANG` setup to support unicode? Also does this happen in gnome-terminal - does the environment differ between the 2 terminals?

**Comment by Felipebros** [comment]
I didn't understand you well. Do you have an example link for this configuration? I am using Xubuntu OS: Ubuntu 18.04.4 LTS x86_64, in the other Xubuntu in VSCode, it only happens in the integrated terminal, only in it, in xfce4-terminal it does not happen.

**Comment by Tyriar** [comment]
Run `echo $LANG` in both terminals, if they don't differ, try diffing `env` to see if something sticks out.

**Comment by Felipebros** [comment]
both `pt_BR.UTF-8`

**Comment by Felipebros** [comment]
VS Code
```
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
LESSCLOSE=/usr/bin/lesspipe %s %s
XDG_MENU_PREFIX=xfce-
LANG=pt_BR.UTF-8
GDM_LANG=pt_BR
DISPLAY=:0.0
ORIGINAL_XDG_CURRENT_DESKTOP=XFCE
GTK_OVERLAY_SCROLLING=0
COLORTERM=truecolor
CHROME_DESKTOP=code-url-handler.desktop
XDG_VTNR=7
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
MANDATORY_PATH=/usr/share/gconf/xubuntu.mandatory.path
GLADE_CATALOG_PATH=:
XDG_SESSION_ID=c2
XDG_GREETER_DATA_DIR=/var/lib/lightdm-data/felipebros
USER=felipebros
GLADE_MODULE_PATH=:
DESKTOP_SESSION=xubuntu
TEXTDOMAINDIR=/usr/share/locale/
DEFAULTS_PATH=/usr/share/gconf/xubuntu.default.path
QT_QPA_PLATFORMTHEME=gtk2
PWD=/home/felipebros/Python/xtractor-crawler
HOME=/home/felipebros
VSCODE_GIT_ASKPASS_NODE=/usr/share/code/code
TEXTDOMAIN=im-config
SSH_AGENT_PID=1601
TERM_PROGRAM=vscode
TERM_PROGRAM_VERSION=1.47.3
QT_ACCESSIBILITY=1
XDG_SESSION_TYPE=x11
XDG_DATA_DIRS=/usr/share/xubuntu:/usr/share/xfce4:/usr/local/share:/usr/share:/var/lib/snapd/desktop:/usr/share
XDG_SESSION_DESKTOP=xubuntu
GLADE_PIXMAP_PATH=:
CLUTTER_BACKEND=x11
VSCODE_GIT_ASKPASS_MAIN=/usr/share/code/resources/app/extensions/git/dist/askpass-main.js
TERM=xterm-256color
SHELL=/bin/bash
XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
IM_CONFIG_PHASE=2
XDG_CURRENT_DESKTOP=XFCE
GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1
SHLVL=2
XDG_SEAT=seat0
LANGUAGE=pt_BR:pt:en
VSCODE_GIT_IPC_HANDLE=/run/user/1000/vscode-git-3a2c015871.sock
GDMSESSION=xubuntu
LOGNAME=felipebros
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
GIT_ASKPASS=/usr/share/code/resources/app/extensions/git/dist/askpass.sh
XDG_RUNTIME_DIR=/run/user/1000
XAUTHORITY=/home/felipebros/.Xauthority
XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session0
XDG_CONFIG_DIRS=/etc/xdg/xdg-xubuntu:/etc/xdg:/etc/xdg
PATH=/home/felipebros/.local/bin:/home/felipebros/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
SESSION_MANAGER=local/felipebros-Latitude-3400:@/tmp/.ICE-unix/1619,unix/felipebros-Latitude-3400:/tmp/.ICE-unix/1619
LESSOPEN=| /usr/bin/lesspipe %s
_=/usr/bin/env
```

**Comment by Felipebros** [comment]
Terminal
```
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
LESSCLOSE=/usr/bin/lesspipe %s %s
XDG_MENU_PREFIX=xfce-
LANG=pt_BR.UTF-8
GDM_LANG=pt_BR
DISPLAY=:0.0
GTK_OVERLAY_SCROLLING=0
COLORTERM=truecolor
XDG_VTNR=7
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
MANDATORY_PATH=/usr/share/gconf/xubuntu.mandatory.path
GLADE_CATALOG_PATH=:
XDG_SESSION_ID=c2
XDG_GREETER_DATA_DIR=/var/lib/lightdm-data/felipebros
USER=felipebros
GLADE_MODULE_PATH=:
DESKTOP_SESSION=xubuntu
DEFAULTS_PATH=/usr/share/gconf/xubuntu.default.path
QT_QPA_PLATFORMTHEME=gtk2
PWD=/home/felipebros
HOME=/home/felipebros
SSH_AGENT_PID=1601
QT_ACCESSIBILITY=1
XDG_SESSION_TYPE=x11
XDG_DATA_DIRS=/usr/share/xubuntu:/usr/share/xfce4:/usr/local/share:/usr/share:/var/lib/snapd/desktop:/usr/share
XDG_SESSION_DESKTOP=xubuntu
GLADE_PIXMAP_PATH=:
CLUTTER_BACKEND=x11
SHELL=/bin/bash
VTE_VERSION=5202
TERM=xterm-256color
XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
XDG_CURRENT_DESKTOP=XFCE
GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1
XDG_SEAT=seat0
SHLVL=1
LANGUAGE=pt_BR:pt:en
WINDOWID=150994947
GDMSESSION=xubuntu
LOGNAME=felipebros
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
XDG_RUNTIME_DIR=/run/user/1000
XAUTHORITY=/home/felipebros/.Xauthority
XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session0
XDG_CONFIG_DIRS=/etc/xdg/xdg-xubuntu:/etc/xdg:/etc/xdg
PATH=/home/felipebros/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
SESSION_MANAGER=local/felipebros-Latitude-3400:@/tmp/.ICE-unix/1619,unix/felipebros-Latitude-3400:/tmp/.ICE-unix/1619
LESSOPEN=| /usr/bin/lesspipe %s
_=/usr/bin/env
```

**Comment by Felipebros** [comment]
diff

```
6a7
> ORIGINAL_XDG_CURRENT_DESKTOP=XFCE
8a10
> CHROME_DESKTOP=code-url-handler.desktop
17a20
> TEXTDOMAINDIR=/usr/share/locale/
21a25,26
> VSCODE_GIT_ASKPASS_NODE=/usr/share/code/code
> TEXTDOMAIN=im-config
22a28,29
> TERM_PROGRAM=vscode
> TERM_PROGRAM_VERSION=1.47.3
29,30c36
< SHELL=/bin/bash
< VTE_VERSION=5202
---
> VSCODE_GIT_ASKPASS_MAIN=/usr/share/code/resources/app/extensions/git/dist/askpass-main.js
31a38
> SHELL=/bin/bash
32a40
> IM_CONFIG_PHASE=2
34a43
> SHLVL=2
36d44
< SHLVL=1
38c46
< WINDOWID=150994947
---
> VSCODE_GIT_IPC_HANDLE=/run/user/1000/vscode-git-3a2c015871.sock
41a50
> GIT_ASKPASS=/usr/share/code/resources/app/extensions/git/dist/askpass.sh
46c55
< PATH=/home/felipebros/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
---
> PATH=/home/felipebros/.local/bin:/home/felipebros/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
49c58,59
< _=/usr/bin/env
\ Nenhum caractere de nova linha no final do arquivo
---
> _=/usr/bin/env
> OLDPWD=/home/felipebros/Python/xtractor-crawler
\ Nenhum caractere de nova linha no final do arquivo
```

**Comment by Tyriar** [comment]
I don't see anything obvious in the environment that would cause it.

You said type but I pasted the text and it worked. Does pasting work for you?

**Comment by Felipebros** [comment]
paste works, just type it doesn't work

**Comment by Tyriar** [comment]
I just tried on Windows/WSL using the Brasil Portuguese language pack and it worked using the windows alt codes here: https://sites.psu.edu/symbolcodes/languages/psu/portuguese/. Even the on screen keyboard lacks altgr though which I'm assuming you're using.

**Comment by Felipebros** [comment]
I'm using `~ + a` or `shift + ``

**Comment by Tyriar** [comment]
I see 'shift+~, a' correctly typing 'â' on Windows, it's probably a Linux desktop environment/IME issue.

**Comment by PublyaDev** [comment]
Having the same issue here. Some solution?

**Comment by eduardosplima** [comment]
Same issue here too, I use Debian with Gnome and my terminal is gnome-terminal. Accents work everywhere, only in vscode integrated terminal doesn't work...

**Comment by Rutrus** [comment]
Same problem here. I use Xubuntu flavour too.
`LANG=en_US.UTF-8` (this is set correctly)
I can't write "tilde" (´) before aeiou for áéíóú in VSCode terminal but I can in other terminals and the whole system.
My Keyboard language is Spanish (default).
Character Ññ works correctly.

**Comment by victormn** [comment]
Same problem here :(

**Comment by rodrigoyoshida** [comment]
I've been dealing with this bug for a long time, used Debian before, now I use Ubuntu 20, always kept vscode up to date but it never worked. I see 4.5k issues opened, wondering if vscode is becoming abandoned.

**Comment by Felipebros** [comment]
We are Vscode or Atom, the community that supports and chooses which technologies will be improved.

**Comment by JoelHernandez343** [comment]
Same problem here. Ubuntu 2020 and code version 1.55.2

**Comment by alvarocarrera** [comment]
Same problem: Ubuntu 20.04.1 LTS and VSCode version: 1.60.0

**Comment by Tyriar** [comment]
I believe this will be fixed in 1.61 with the fix for https://github.com/microsoft/vscode/issues/82489#issuecomment-914255300
